wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -3,24 +3,21 @@ import queryString from 'query-string';
 import io from "socket.io-client";
 import { Redirect } from "react-router-dom";
 
-import './Lobby.css';
-
-// change to http://localhost:5000 for local development
-const ENDPOINT  = 'http://localhost:5000';
+//Components
+import NoMatch from 'components/No Match/noMatch'; 
+import Swiping from 'views/swiping/swiping-container';
 
 // change to https://wewatch-server.herokuapp.com/ for production deployment
-// const ENDPOINT = 'https://wewatch-server.herokuapp.com/';
-
-
+const ENDPOINT = 'http://localhost:5000';
 
-let socket;
+let socket = io(ENDPOINT);
 
 const Lobby = ({location}) => {
     const [name, setName] = useState('');
     const [hostName, setHostName] = useState('');
     const [roomId, setRoomId] = useState('');
     const [users, setUsers] = useState([]);
-    const [goSwipe, setGoSwipe] = useState(false);
+    const [page, setPage] = useState('Lobby')
```

Comment:
Do you think we should add some comments to this logic? Just because it may not be trivial.